docs/source/tutorials/artifact.rst

```diff
@@ -0,0 +1,201 @@
+========
+Artifact
+========
+
+The :class:`~vivarium_public_health.dataset_manager.artifact.Artifact` in vivarium_public_health provides an interface
+to interact with the data artifacts that can be used as the underlying storage for input data into a simulation. We'll
+go through how to view, delete, and write data to an artifact using the tools provided by
+:class:`~vivarium_public_health.dataset_manager.artifact.Artifact`. You'll access data in the artifact through keys,
+mirroring the underlying hdf storage of artifacts.
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+Creating an artifact
+---------------------
+To view an existing hdf file via the :class:`~vivarium_public_health.dataset_manager.artifact.Artifact` tools, we'll
+create a new artifact. We can print the resulting artifact to view the tree structure of the keys
+in our artifact. We'll use our test artifact to illustrate:
+
+.. code-block:: python
+
+    from vivarium_public_health.dataset_manager import Artifact
+
+    art = Artifact('test_artifact.hdf')
+    print(art)
+
+::
+
+    Artifact containing the following keys:
+    metadata
+            keyspace
+            locations
+            versions
+    population
+            age_bins
+            structure
+            theoretical_minimum_risk_life_expectancy
+
+Now we have an :class:`~vivarium_public_health.dataset_manager.artifact.Artifact` object, which we can use to interact
+with the data stored in the hdf file with which we created it.
+
+Optionally, we can specify filter terms on the artifact when we create it, which will be applied to filter all data
+that we load out of the artifact. For example, if we were only interested in data for years after 2005, we could add a
+filter term to our artifact that would filter all data with a ``year_start`` column. Any data in our artifact that does
```

Comment:
It might be useful to reference df.query or pd.read_hdf(..., where=).  Or hide that. but some analogy to filtering a dataframe might be useful.